### PR TITLE
Contribution guide ✨

### PR DESCRIPTION
**Pull request by tbarn**
### What kind of change does this PR introduce?

Docs update

### What is the current behavior?

There are no contribution docs right now

### Does this PR introduce a breaking change?

Nope

What changes might users need to make in their application due to this PR?

### Other information

I'd love feedback on if this is welcoming to an outside contributor to Stoplight. I tried to add some resources to help get people up to speed and explain some of the things we do, like `yarn commit`, that might be outside of some contributor's past experiences.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1 +1,93 @@
...
+
+At Stoplight, we want contributing to Spectral to be an enjoyable and educational project for everyone. Contributions go beyond commits in pull requests. We are excited to receive contributions in the form of:
+
+- feature ideas
+- pull requests
+- triaging issues
+- reviewing pull requests
+- implementations of Spectral in your own projects
+- blog posts, talks referencing the project, tweets
+- and much more!
```
Comment (rowa97) [inline CONTRIBUTING.md]: not sure you want to leave this open-ended like that

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1 +1,93 @@
...
+
+If it is related to Spectral, we consider it a contribution. 
+
+If you are new to contributing to open source, GitHub has created a helpful guide with lots of resources [here](https://opensource.guide/how-to-contribute/). If you want more help, post in our [Community forum](https://community.stoplight.io/c/open-source) or send us an email to [support@stoplight.io](mailto:support@stoplight.io). We are happy to help you out there.
```
Comment (rowa97) [inline CONTRIBUTING.md]: or send an email to

Comment (rowa97) [reply CONTRIBUTING.md]: or send us an email

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1 +1,93 @@
...
+- blog posts, talks referencing the project, tweets
+- and much more!
+
+If it is related to Spectral, we consider it a contribution. 
+
+If you are new to contributing to open source, GitHub has created a helpful guide with lots of resources [here](https://opensource.guide/how-to-contribute/). If you want more help, post in our [Community forum](https://community.stoplight.io/c/open-source) or send us an email to [support@stoplight.io](mailto:support@stoplight.io). We are happy to help you out there. 
+
+We want to encourage everyone to be welcoming to newcomers and encourage new contributors from all backgrounds.
+
+Also, to help create an environment where anyone could potentially be welcome to contribute, we have a Code of Conduct that applies to the project and adjacent spaces related to Spectral.
```
Comment (rowa97) [inline CONTRIBUTING.md]: Also is unnecessary

File: CONTRIBUTING.md (hunk 1 above)
Comment (tbarn) [reply CONTRIBUTING.md]: I do! The next line makes it even clearer. 

> If it is related to Spectral, we consider it a contribution.

**Pull request merged**